frame/fast-unstake/src/lib.rs

```diff
@@ -192,6 +201,29 @@ pub mod pallet {
 				return Weight::from_ref_time(0)
 			}
 
+			// Temporary for migration.
```

Comment:
Could also be a migration for clarity and posterity. A bit more work, but probably worth it. Then you won't need to make an extra PR with more changes to this code either.